tests/storages_tests/test_storages.py

```diff
@@ -161,70 +164,44 @@ def test_get_study_id_from_name_and_get_study_name_from_id(storage_mode: str) ->
             storage.get_study_name_from_id(study_id + 1)
 
 
+# TODO(gen740): write a test create_study directions and get_study_directions
```

Comment:
Is this TODO comment addressed, or left as a follow-up?

Reply:
Oh..., I forgot to delete this comment.